### PR TITLE
Final Changes

### PR DESCRIPTION
**Pull request by ayushjain01**
Completed all tasks - 
image

Sorry! will make smaller PRs next time onwards. 😅

Home page currently displays all items in descending order, will modify it to display only one article every folder.

### CODE REVIEW

**Comment by ayushjain01** [comment]
Previews - https://ayushjain01.github.io/learn-pip-trends/

File: _blog/1-whats next.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+layout: article
+title: learn-pip-trends - What's Next?
+permalink: /blogs/whats-next.html
+key: blogs/whats-next
+cover: /assets/images/blogs/Blog - 1.png
```
Comment (tankala) [inline _blog/1-whats next.md]: Don't use spaces for file names. In coding, we always avoid spaces it applies to variable names, function names & file names also.

File: _config.yml [hunk 2]
```diff
@@ -1,56 +1,56 @@
...
 title   : learn-pip-trends
-description: > # this means to ignore newlines until "Language & timezone"
+tagline : Learn everything about Python
+description: > 
   Learn everything about Python
+author : Ashok Tankala
+twitter:
+  username: piptrends
+  card: summary
+facebook:
```
Comment (tankala) [inline _config.yml]: What is the advantage of facebook here. Check once.

File: _config.yml [hunk 3]
```diff
@@ -1,56 +1,56 @@
...
-  googleplus: # "user_id"   the last part of your profile url, e.g. https://plus.google.com/u/0/user_id
-  telegram  : # "user_name" the last part of your profile url, e.g. https://t.me/user_name
-  medium    : # "user_name" the last part of your profile url, e.g. https://medium.com/user_name
-  zhihu     : # "user_name" the last part of your profile url, e.g. https://www.zhihu.com/people/user_name
-  douban    : # "user_name" the last part of your profile url, e.g. https://www.douban.com/people/user_name
-  linkedin  : # "user_name" the last part of your profile url, e.g. https://www.linkedin.com/in/user_name
-  github    : # "user_name" the last part of your profile url, e.g. https://github.com/user_name
-  npm       : # "user_name" the last part of your profile url, e.g. https://www.npmjs.com/~user_name
+  email     : ashok.tankala@gmail.com
+  blog      : https://blog.tanka.la/
```
Comment (tankala) [inline _config.yml]: we need to change these also. We will use blog.piptrends.com, ashok@piptrends.com

File: _config.yml [hunk 4]
```diff
@@ -145,7 +138,7 @@ pageview:
 ## => Search
 ##############################
 search:
-  provider: default # "default" (default), false, "google", "custom"
+  provider: false # "default" (default), false, "google", "custom"
```
Comment (tankala) [inline _config.yml]: We will keep search

File: _gettingstarted/1-install-python.md [hunk 5]
```diff
@@ -3,8 +3,17 @@ layout: article
...
 aside:
   toc: true
+article_header:
+  type: overlay
+  theme: dark
+  background_color: '#203028'
+  background_image:
+    gradient: 'linear-gradient(135deg, #2BF29F, #FFFF00)'
+    src: /docs/assets/images/cover3.jpg
+date: 2023-08-01
```
Comment (tankala) [inline _gettingstarted/1-install-python.md:20]: We need to add keywords & rest of the SEO-related stuff. SEO gem we added for that purpose only.

**Review by tankala — CHANGES_REQUESTED** [review]
Based on the given comments look rest of the code too

File: _blog/1-whats next.md (hunk 1 above)
Comment (ayushjain01) [reply _blog/1-whats next.md]: Fixed

File: _config.yml (hunk 2 above)
Comment (ayushjain01) [reply _config.yml]: This is for share preview(Open Graph) on Facebook.

File: _config.yml (hunk 3 above)
Comment (ayushjain01) [reply _config.yml]: Changed!

File: _gettingstarted/1-install-python.md (hunk 5 above)
Comment (ayushjain01) [reply _gettingstarted/1-install-python.md:20]: Added description, keywords, tags, and image

File: _config.yml (hunk 4 above)
Comment (ayushjain01) [reply _config.yml]: Sorry, had disabled it for testing, forgot to enable it back. 
Fixed it now

**Comment by ayushjain01** [comment]
Changes committed

**Review by tankala — APPROVED** [review]

**Pull request merged**